docs/mevblocker/concepts/Order Flow Auction/order-flow-auction.md

```diff
@@ -0,0 +1,51 @@
+---
+sidebar_position: 3
+---
+# MEV Blocker OFA
+
+MEV Blocker's Transaction Flow can be broken down into 6 major steps. In the picture below you can find a general overview of the flow of a transaction.
+
+![The Best Protection Under the Sun](/img/mevblocker/mevblocker_ofa.png)
+
+## User submits the transactions
+
+This is the first step, where the user creates the transaction:
+
+1. If you are a normal user in which you have 100% control of all settings, you must first have installed the custom MEV Blocker RPC in your wallet. Learn more about how to do it here
```

Comment:
Where is the link to "how to do it here" ?

Reply:
Need to delete